lib/Module/Starter/Smart.pm

```diff
@@ -170,11 +170,12 @@ sub create_t {
 }
 
 sub _create_t {
-    my $self = shift;
+    my $self     = shift;
+    my $testdir  = @_ == 2 ? 't' : shift;
```

Comment:
I don't know. From the point of view of the Smart plugin it's definitely not optimal, but for the purposes of Simple the argument order makes sense.